client/artifact.py

```diff
@@ -62,7 +62,11 @@ def __init__(self, account, repo):
         url = self.__METAURL.format(account, repo)
         self.metadata = json.loads(_getremotedata(url))
         self.version = self.metadata['name']
-        self.pkgurl = self.metadata['assets'][0]['browser_download_url']
+
+        if self.metadata['assets']:
```

Comment:
I think you want `'assets' in self.metadata` ?  this will raise if `assets` isn't in the dict.

Reply:
Sure